src/ocm/reducers/clusters/clustersSlice.ts

```diff
@@ -3,13 +3,15 @@ import { Cluster } from '../../../common';
 import { handleApiError } from '../../api/utils';
 import { ResourceUIState } from '../../../common';
 import { ClustersAPI } from '../../services/apis';
+import { ocmClient } from '../../api';
 
 export const fetchClustersAsync = createAsyncThunk<Cluster[] | void>(
   'clusters/fetchClustersAsync',
   async () => {
     try {
       const { data } = await ClustersAPI.list();
-      return data.filter((cluster) => cluster.kind === 'Cluster');
+      const isOcm = !!ocmClient;
+      return data.filter((cluster) => (isOcm ? cluster.kind === 'Cluster' : true));
```

Comment:
@jgyselov why do we need this?

Reply:
Because otherwise a cluster transformed to day2 (`cluster.kind = "AddHostsCluster"`) would disappear from the clusters list. I'd say disappearing clusters is undesired behaviour. @jkilzi